parachain/pallets/inbound-queue/Cargo.toml

```diff
@@ -63,14 +61,18 @@ std = [
     "snowbridge-router-primitives/std",
     "ethabi/std",
     "xcm/std",
+    "xcm-builder/std",
```

Comment:
`xcm-builder` is required for running `cargo test -p snowbridge-inbound-queue --features runtime-benchmarks`